tests/v2/validation/deleting/README.md

```diff
@@ -0,0 +1,34 @@
+# Deleting
+
+## Getting Started
+You can find specific tests by checking the test file you plan to run. An example is `-run ^ TestClusterDeleteTestSuite/TestDeletingRKE2K3SCluster$`
+In your config file, set the following:
+```json
+"rancher": { 
+  "host": "rancher_server_address",
+  "adminToken": "rancher_admin_token",
+  "userToken": "your_rancher_user_token",
+  "clusterName": "cluster_to_run_tests_on",
+  "insecure": true/optional,
+  "cleanup": false/optional,
+}
+```
+
+Typically, a cluster with the following 3 pools is used for testing:
+```yaml
+{
+  {
+    ControlPlane: true,
+    Quantity:     2,
+  },
+  {
+    Etcd:     true,
+    Quantity: 3,
+  },
+  {
+    Worker:   true,
+    Quantity: 2,
+  },
+}
+  ```
```

Comment:
I think we can remove this from the deleting, another small nit I think config is updated for scaling README